src/index.js

```diff
@@ -15,73 +15,71 @@ const plugin = ({ term, display, actions }) => {
 
     //fall back on the ts prefix
     if (!match) {
-        match = term.match(/^tsu?\s(.*)/);
+        match = term.match(/^tsu?\s*($|.*)/);
```

Comment:
Can you explain why you changed the regex?
I'm under the impression that the first `*` will catch everything, so the change after it won't affect anything.

![screenshot 2017-11-02 09 04 09](https://user-images.githubusercontent.com/1698042/32327420-d352a2b4-bfac-11e7-98a0-3671013dc6df.png)

![screenshot 2017-11-02 09 04 19](https://user-images.githubusercontent.com/1698042/32327424-d5411fd8-bfac-11e7-9ae5-a1c2b25e99b2.png)

Reply:
Sure - the old regex requires at least one space, i.e. `ts `, the new regex works with plain `ts`

Reply:
Then I guess `^tsu?\s?(.*)` would be enough to handle that case

Reply:
Indeed, let me quickly adapt